src/Switches.h

```diff
@@ -95,6 +96,33 @@ class SwitchOnCycleDuration {
 		g_cycleStartTime = nextCycleStartTime;
 	}
 };
+
+//=============================================================================
+class SwitchOnDummySend {
+public:
+	// dummy send between two cycles starting point and take starting point of next cycle
+
+	inline void execute(int ifd, int buffer_size) {
+		uint8_t buffer[buffer_size];
+		TicksTime nextCycleStartTime = g_cycleStartTime + g_pApp->m_const_params.cycleDuration;
+		TicksTime nextDummySendTime = g_cycleStartTime + g_pApp->m_const_params.dummySendCycleDuration;
+		while (!g_b_exit) {
+			if (TicksTime::now() >= nextCycleStartTime) {
+				break;
+			}
+
+			if (TicksTime::now() > nextDummySendTime) {
+				sendto(ifd, buffer, sizeof(buffer), DUMMY_SEND_FLAG, (struct sockaddr*)& (g_fds_array[ifd]->server_addr), sizeof(struct sockaddr));
```

Comment:
we should consider using a flag which is not supported by OS so this frame gets dropped and does not get queued by OS on TCP and UDP flows.